src/index.js

```diff
@@ -69,10 +69,10 @@ ${unused.join(`\n`)}`);
   }
 
   _getFileDepsMap(compilation) {
-    const fileDepsBy = compilation.fileDependencies.reduce((acc, usedFilepath) => ({
-      ...acc,
-      [usedFilepath]: usedFilepath,
-    }), {});
+    const fileDepsBy = compilation.fileDependencies.reduce((acc, usedFilepath) => {
+      acc[usedFilepath] = usedFilepath;
```

Comment:
I think @ljharb's suggestion of using Set is right move long term. Just wanted fast rebuilds back first so thanks for merging!